### ISSUE TITLE
Survey the UI for accesibility issues

### ISSUE THREAD

**Issue by rdubrock** [grafana/synthetic-monitoring-app#253]
We should identify UI accessibility issues in the app. The issues should be captured in a github ticket

**Comment by jewbetcha** [comment]
Handled in https://github.com/grafana/synthetic-monitoring-app/pull/406
